src/Stryker.Core/Stryker.Core/Reporters/ReporterFactory.cs

```diff
@@ -15,19 +16,33 @@ public static IReporter Create(StrykerOptions options)
 
         private static IDictionary<Reporter, IReporter> CreateReporters(StrykerOptions options)
         {
+            var jsonReporter = new JsonReporter(options);
+            var htmlReporter = new HtmlReporter(options, jsonReporter);
             return new Dictionary<Reporter, IReporter>
             {
                 { Reporter.ConsoleProgressDots, new ConsoleDotProgressReporter() },
                 { Reporter.ConsoleProgressBar, CreateProgressReporter() },
                 { Reporter.ConsoleReport, new ConsoleReportReporter(options) },
-                { Reporter.Json, new JsonReporter(options) }
+                { Reporter.Json, jsonReporter },
+                { Reporter.Html, htmlReporter }
             };
         }
 
         private static IEnumerable<IReporter> DetermineEnabledReporters(IEnumerable<Reporter> enabledReporters, IDictionary<Reporter, IReporter> possibleReporters)
         {
-            return enabledReporters.Contains(Reporter.All) ? possibleReporters.Values :
-                possibleReporters.Where(reporter => enabledReporters.Contains(reporter.Key)).Select(reporter => reporter.Value);
+            if (enabledReporters.Contains(Reporter.All) is var all && all || enabledReporters.Contains(Reporter.Html))
```

Comment:
I don't see why we should limit users in what kind of report they might want. They might want to use it in some automated process, someone might want to build a build server plugin that displays the report from the json et cetera. We also have no way of knowing if someone might actually already be using it. It would be a major breaking change to remove it.

Reply:
Good point, but still I think it's not good design to disable the Json reporter when the Html reporter is enabled. What harm would it cause if both are enabled? The json is generated two times but this is by choice of the user. I would still like to see them both enabled at once.